ui/apps/platform/src/utils/searchUtils.test.ts

```diff
@@ -380,7 +380,7 @@ describe('searchUtils', () => {
                 expect(offsetParam).not.toBe('');
                 expect(typeof offsetParam).toBe('string');
                 const offset = parseInt(offsetParam as string, 10);
-                expect(offset % page).toBe(0);
+                expect(offset % pageSize).toBe(0);
```

Comment:
Good catch. Because it is hard to see how `[3, 10]` and `[10, 3]` passed for this assertion, is it possible that the logic to compute `offset` was also incorrect somehow?

Reply:
So I'm fairly sure this is due to calculating the `page - 1` inside the function call instead of before it like we did prior to this change. It looks like the test cases just happened to line up so that the assertions all passed. Once I saw the failures, I realized that logically what this was trying to test didn't even make sense.